cli/dts/lib.deno.unstable.d.ts

```diff
@@ -421,11 +421,25 @@ declare namespace Deno {
       | "es2019"
       | "es2020"
       | "esnext";
-    /** List of names of type definitions to include. Defaults to `undefined`.
+    /** List of names of type definitions to include when type checking.
+     * Defaults to `undefined`.
      *
      * The type definitions are resolved according to the normal Deno resolution
-     * irrespective of if sources are provided on the call. Like other Deno
-     * modules, there is no "magical" resolution. For example:
+     * irrespective of if sources are provided on the call. In addition, unlike
+     * passing the `--config` option on startup, there is no base to resolve
+     * relative specifiers, so the specifiers here have to be fully qualified
+     * URLs or paths.  For example:
```

Comment:
Interesting point.  The current challenge is that we have to effectively set a "cwd" for the relative paths in types, because tsc will request relative modules relative to the "cwd" we provide it, unlike modules where we are provided with the referrer and we get to fully resolve the module however we see fit.  It is similar to import-maps.

At the point where we determine the cwd in tsc, it is hard to figure out the root of the `Deno.emit()` at the moment.  We could use the real `cwd` but that means the relative point of `Deno.emit()` and this would/could be different.  What I would like to do is land it with this restriction at the moment and consider figuring out a way to make it relative as an enhancement?